### PR TITLE
Recreate template, having substitution problems

### PR DESCRIPTION
**Pull request by DerekRoberts**
There have been substitution errors (`$` and `*`) sending the completed template between action steps.  This recreates the template, sidestepping our issue.  (a full fix would be better)

### CODE REVIEW

**Comment by DerekRoberts** [comment]
@paulushcgcj this is tested working in https://github.com/bcgov/nr-forest-client/pull/228 (Dependabot).  After merge we can update nr-forest-client to use the upcoming release (1.0.0).

**Review by paulushcgcj — APPROVED** [review]

**Comment by paulushcgcj** [comment]
I'll wait for this to be merged and released before moving on with https://github.com/bcgov/nr-forest-client/pull/228

**Review by mishraomp — APPROVED** [review]
LGTM

**Pull request merged**
